templates/shared/user/name.tmpl

```diff
@@ -1,5 +1 @@
-{{if .FullName}}
-	{{.FullName}} <span>(<a class="text primary" href="{{.HomeLink}}">{{.Name}}</a>)</span>
-{{else}}
-	<a class="text primary" href="{{.HomeLink}}">{{.Name}}</a>
-{{end}}
+<a class="text muted" href="{{.HomeLink}}">{{if .FullName}}{{.FullName}} ({{.Name}}){{else}}{{.Name}}{{end}}</a>
```

Comment:
For consistency, the format should be `Name (FullName)`

Reference:

1. GitHub

<details>

![image](https://github.com/go-gitea/gitea/assets/2114189/c818bcae-095d-48f7-b1f9-17dde170e7dc)

</details>

2. `search_name.tmpl`

Reply:
Done